iphone/Maps/UI/PlacePage/Components/PlacePageHeader/PlacePageHeaderViewController.swift

```diff
@@ -50,7 +50,35 @@ extension PlacePageHeaderViewController: PlacePageHeaderViewProtocol {
     }
   }
 
-  func setTitle(_ title: String) {
-    titleLabel?.text = title
+  func setTitle(_ title: String?, secondaryTitle: String?) {
+    guard let title = title,
+          let titleFont = StyleManager.shared.theme?.fonts.medium20 else {
+      titleLabel?.attributedText = nil
+      return
+    }
+
+    let titleAttributes: [NSAttributedString.Key: Any] = [
+      .font: titleFont,
+      .foregroundColor: UIColor.blackPrimaryText()
```

Comment:
What about the dark mode?

Reply:
Will check it, but inside it uses `StyleManager.shared.theme`.